contracts/pair_stable_bluna/src/contract.rs

```diff
@@ -1257,11 +1258,33 @@ pub fn query_pending_reward(deps: Deps, _env: Env, user: String) -> StdResult<As
         Decimal256::zero()
     };
 
+    let accrued_rewards: AccruedRewardsResponse = deps.querier.query_wasm_smart(
+        config.bluna_rewarder,
+        &BAssetRewardQueryMsg::AccruedRewards {
+            address: env.contract.address.to_string(),
```

Comment:
is the pair_stable_bluna contract is a holder for the anchor basset reward contract?

Reply:
BLUNA_REWARD_HOLDER is only used to keep claimed rewards for further distribution among liquidity providers

Reply:
BLUNA_REWARD_HOLDER is only used to keep claimed rewards for further distribution among liquidity providers